src/Timers.cpp

```diff
@@ -198,6 +224,13 @@ std::vector<Timer> Timers::LoadTimers()
   return timers; 
 }
 
+bool Timers::findTagInTimerTags(std::string tag, std::string tags)
```

Comment:
You should decide wheter you want to start method names with a smal or a capital letter. Currently, there is both.

Reply:
Done